x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/relevance_tuning_form/relevance_tuning_form.scss

```diff
@@ -0,0 +1,9 @@
+.relevanceTuningForm {
```

Comment:
I don't know what the answer is to that. I haven't put it into any of my `.scss` files, largely because lint doesn't require it.

Looking at the rest of Kibana, it looks like *no* scss files have a license header.

Reply:
Hah super interesting! I guess I was being extra when putting those in. Let's just do whatever I guess, no lint rule = no problem :trollface: